molecularnodes/blender/nodes.py

```diff
@@ -237,6 +250,16 @@ def material_default():
 
     return bpy.data.materials[mat_name]
 
+def MN_micrograph_material():
+    """
+    Append MN_micrograph_material to the .blend file it it doesn't already exist, 
+    and return that material.
+    """
+
+    mat_name = 'MN_micrograph_material'
```

Comment:
Additionally I've started moving towards an emphasis on 'human readable' names for node groups and materials etc.

It's not _ideal_ for programming them, but they are only ever reference as strings anyway. The default material is called 'MN Default' now - so I think going forward we try to keep things named for people searching and reading them. Maybe `MN Micrograph`, as it is a material group so we won't need to reiterate that

Reply:
This is only really for user-facing components like node groups, internal components I think we stick to cleaner naming but when users are searching or reading, not having the underscores helps